tests/conftest.py

```diff
@@ -1592,6 +1603,49 @@ def additional_testsuite_properties(record_testsuite_property, pytestconfig):
     # add markers as separated property
     markers = ocsci_config.RUN["cli_params"].get("-m", "").replace(" ", "-")
     record_testsuite_property("rp_markers", markers)
+    if is_dr_scenario():
```

Comment:
@OdedViner , looks like this block of code is mostly repetition of the block above https://github.com/red-hat-storage/ocs-ci/pull/9956/files#diff-715cca595e4d8f8493e6afe4e6605ebb2b8f91d3cb1be10495f2ec06e3826261R218, is it possible to optimize this ?

Reply:
done